Automattic/jetpack
# Fix/fontfamily attribute

Fixes #38311

## Proposed changes:

* Check if the block attribute `fontFamily` is a string before setting the font family.

### Other information:
This change allows other blocks to use a different data type as a `fontFamily`. 


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Install a block that uses a `fontFamily` attribute that is not a string. see https://wordpress.org/plugins/easy-quotes/ as an example
* Add the block to a post
* Note that the post cannot be saved.
